lib/routes/routeBackbeat.js

```diff
@@ -1289,7 +1289,10 @@ function routeBackbeat(clientIP, request, response, log) {
                     [request.query.operation](request, response, log, next);
             }
             const versioningConfig = bucketInfo.getVersioningConfiguration();
-            if (!versioningConfig || versioningConfig.Status !== 'Enabled') {
+            // The following makes sure that only replication destination-related operations (non-GET requests)
+            // target buckets with versioning enabled.
+            // This allows lifecycle operations (which use GET requests) to work on non-versioned buckets.
+            if (request.method !== 'GET' && (!versioningConfig || versioningConfig.Status !== 'Enabled')) {
```

Comment:
Replication is supposed to work on any versioned bucket : so it _must_ work as well on version-suspended buckets
```suggestion
            if (request.method !== 'GET' && (!versioningConfig || versioningConfig.Status === 'Disabled')) {
```

Reply:
In CRR replication both source and replication have to be version enabled. I will keep enforcing it.